octoprint_mrbeam/iobeam/laserhead_handler.py

```diff
@@ -294,6 +306,17 @@ def get_correction_settings(self):
     def get_current_used_lh_model_id(self):
         return self._current_used_lh_model_id
 
+    def is_current_used_lh_model_supported(self):
+        """Checks if the current used laser head model is supported or not.
+
+        Returns:
+            boolean: True if the current used laser head model is supported, False otherwise
+        """
+        supported = SUPPORTED_LASERHEADS.get(self.get_current_used_lh_model_id()) is not None and self._plugin.get_model_id() in SUPPORTED_LASERHEADS.get(self.get_current_used_lh_model_id())
```

Comment:
Maybe could be simplified to:
```suggestion
        supported = self._plugin.get_model_id() in SUPPORTED_LASERHEADS.get(self.get_current_used_lh_model_id(), [])
```